src/main/java/uk/gov/hmcts/reform/civil/stateflow/transitions/ClaimSubmittedTransitionBuilder.java

```diff
@@ -42,7 +42,7 @@ void setUpTransitions(List<Transition> transitions) {
             .moveTo(TAKEN_OFFLINE_BY_STAFF, transitions).onlyWhen(takenOfflineByStaffBeforeClaimIssued, transitions)
             .moveTo(CLAIM_ISSUED_PAYMENT_FAILED, transitions).onlyWhen(paymentFailed, transitions)
             .moveTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_ONE_V_ONE_SPEC, transitions).onlyWhen(
```

Comment:
Does it not work by moving this check above 
            .moveTo(TAKEN_OFFLINE_BY_STAFF, transitions).onlyWhen(takenOfflineByStaffBeforeClaimIssued, transitions)